hack/import-calendar.py

```diff
@@ -96,6 +96,7 @@ def read_calendar(cal):
                 'label': str(event['summary']),
                 'where': format_location_html(event_location),
                 'description': description,
+                'timestamp': event_time,
```

Comment:
`event_time` is not a timestamp but a `datetime.datetime`.

Reply:
The output in the YAML file is shaped like this (Python string formatter coerces it into a timestamp):

```
  timestamp: 2024-04-04 15:00:00+00:00
```

I've been planning to provide localized time with client-side JS, having the timestamp in a format that I can reliably parse back will be invaluable.

Reply:
Yes, I think we can rely on the Python string formatter to always format DateTime in a way that I can pass to Time.parse in any language down the line. A unix timestamp would also suffice for my needs here.

Reply:
Thanks for your review! 🙇‍♂️